manager/manager.go

```diff
@@ -432,7 +432,7 @@ func (m *Manager) Run(parent context.Context) error {
 		grpc.WithTransportCredentials(m.config.SecurityConfig.ClientTLSCreds),
 	}
 
-	cs := raftpicker.NewConnSelector(m.RaftNode, proxyOpts...)
+	cs := raftpicker.NewConnSelector(ctx, m.RaftNode, proxyOpts...)
```

Comment:
The raft node is destroyed before this context is cancelled, so I think the goroutine inside the raft picker could panic on shutdown when it tries to call `LeaderAddr`.